backend/config/lighthouse.php

```diff
@@ -210,7 +210,7 @@
     |
     */
 
-    'debug' => \GraphQL\Error\Debug::INCLUDE_DEBUG_MESSAGE | \GraphQL\Error\Debug::INCLUDE_TRACE,
+    'debug' => env('LIGHTHOUSE_DEBUG', \GraphQL\Error\DebugFlag::INCLUDE_DEBUG_MESSAGE | \GraphQL\Error\DebugFlag::INCLUDE_TRACE),
```

Comment:
This makes me wonder if we should have documentation somewhere for `.env` variables, unless it already exists?